# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/android/nsWindow.cpp
# Commit: 1fa3e699ce84
# Full Hash: 1fa3e699ce84a2c7562e25e2e3a6c1194372042f
# Author: David Parks <davidp99@gmail.com>
# Date: 2024-07-04 16:09:52
# Regressor Bug: 1893119
# File Overlap Count: 1
# Description:
#   Bug 1893119: Part 2 - Make C++ callers of nsIDragService::GetCurrentSession pass a widget r=gstoll,win-reviewers,geckoview-reviewers,m_kato
#   
#   Content process callers must call nsIDragService::GetCurrentSession with the
#   correct widget under the drag.  For all callers, it is a sanity check.
#   
# ==============================================================================

diff -r 21e7f9ac67f8 -r 1fa3e699ce84 widget/android/nsWindow.cpp
--- a/widget/android/nsWindow.cpp	Thu Jul 04 07:48:03 2024 +0000
+++ b/widget/android/nsWindow.cpp	Thu Jul 04 07:48:03 2024 +0000
@@ -2705,8 +2705,7 @@
     nsDragService::SetDropData(dropData);
   }
 
-  nsCOMPtr<nsIDragSession> dragSession;
-  dragService->GetCurrentSession(getter_AddRefs(dragSession));
+  nsCOMPtr<nsIDragSession> dragSession = dragService->GetCurrentSession(this);
   if (dragSession) {
     switch (message) {
       case eDragOver: